api/types/accesslist/convert/v1/accesslist_test.go

```diff
@@ -184,11 +184,20 @@ func TestFromProtoNils(t *testing.T) {
 		_, err := FromProto(msg)
 		require.NoError(t, err)
 	})
+
+	t.Run("member_count", func(t *testing.T) {
+		msg := ToProto(newAccessList(t, "access-list"))
```

Comment:
What does this test tests?
Only if FromProto doesn't return any error? It still doesn't validate if the members count was propagated from and to protons

Reply:
This effectively makes sure that the member count being `nil` doesn't break/panic. The RoundTrip test function tests the propagation.